api/services/Flairs.js

```diff
@@ -261,6 +263,13 @@ exports.makeNewCSSClass = function (previous_flair, new_addition, subreddit) {
   if (new_addition === 'involvement') {
     return previous_flair.replace(/( |$)/, '1$1');
   }
+  if (extraFlair.includes(new_addition)) {
```

Comment:
I don't remember. I think we're still using Babel 5, and I forget how that works with regard to polyfills.